### PR TITLE
Prep release

### PR DESCRIPTION
**Pull request by olipyskoty**
**Contributing to Twilio**

> All third party contributors acknowledge that any contributions they provide will be made under the same open source license that the open source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### JIRA link(s):

- [VIDEO-10660](https://issues.corp.twilio.com/browse/VIDEO-10660)

### Description

This PR properly fixes the bug where the chat window is to be open by default unless the environment variable `DISABLE_CHAT=true`. This PR also updates the `CHANGELOG` and version numbers to prep for the release.

### Before review
* [ ] Added unit tests if necessary
* [ ] Updated affected documentation
* [ ] Verified locally with all effected platforms
* [x] Manually sanity tested running locally
* [ ] Included screenshot as PR comment (if needed)
* [x] Ready for review

### Before merge
* [ ] Got one or more +1s
* [ ] Re-tested if necessary

### CODE REVIEW

File: CHANGELOG.md
Comment (timmydoza) [inline CHANGELOG.md:9]:
```suggestion
- Fixes an issue where the host could not create more than one stream. [#116](https://github.com/twilio/twilio-live-interactive-video/pull/116)
```

**Review by timmydoza — CHANGES_REQUESTED** [review]
Looks good, just one comment.

@timrozum - I think some things need to be updated on the iOS side with the new version number in preparation for the release. Do you just want to push up your own commit to this branch with those changes?

**Comment by timrozum** [comment]
@timmydoza I just as well push the iOS version bump to the existing iOS branch and merge it. I will use 1.1.0. We should hold off on Github release until both branches are merged.

In the future I think it would work better if we did this for releases:

1. Merge all PRs for code changes we want in the release to main branch.
2. Create a new PR just for the release that includes version bumps and any final changelog changes.

Reason being is that releases are higher level than any single improvement/feature PR. I think even in the release we are doing today there is a bug fix for a PR that was merged a while ago for example.

**Pull request merged**
